README.md

```diff
@@ -1,2 +1,237 @@
+== Adding stacks from index https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+==== Adding stack nodejs-loopback 0.2.2
+==== Excluding stack swift 0.2.5
+==== Excluding stack node-red 0.1.1
+==== Excluding stack python-flask 0.2.2
+==== Excluding stack starter 0.1.2
+==== Excluding stack java-microprofile 0.2.24
+==== Adding stack java-spring-boot2 0.3.26
+==== Excluding stack kitura 0.2.5
+==== Adding stack java-openliberty 0.2.2
+==== Excluding stack nodejs 0.3.4
+==== Adding stack nodejs-express 0.4.3
+== fetching https://github.com/kabanero-io/collections/releases/download/0.6.0/kabanero-index.yaml
+== Adding stacks from index https://github.com/kabanero-io/collections/releases/download/0.6.0/kabanero-index.yaml
+==== Adding stack java-microprofile 0.2.25
+==== Excluding stack java-openliberty 0.2.1
+==== Excluding stack java-spring-boot2 0.3.23
+==== Excluding stack nodejs-express 0.2.8
+==== Excluding stack nodejs-loopback 0.1.8
+==== Excluding stack nodejs 0.3.1
+ == Running post_build.d scripts
+ ==== Running build_nginx.sh
+ == Done post_build.d scripts
+ ```
+
+ Switch to the `assets` directory that is generated by the build to find the stack hub index file in two formats, `default-index.yaml` and `default-index.json`. The JSON file format is required if you are developing applications by using Codewind.
+
+ Open the `default-index.yaml` file to view the consolidated information for each stack in your configuration. For example, the last stack in the list is `nodejs-express`:
+
+ ```
+ - default-template: simple
+  description: Express web framework for Node.js
+  id: nodejs-express
+  image: docker.io/appsody/nodejs-express:0.4.3
+  language: nodejs
+  license: Apache-2.0
+  maintainers:
+  - email: vieuxtech@gmail.com
+    github-id: sam-github
+    name: Sam Roberts
+  name: Node.js Express
+  requirements:
+    appsody-version: '>= 0.2.7'
+    docker-version: '>= 17.09.0'
+  src: https://github.com/appsody/stacks/releases/download/nodejs-express-v0.4.3/incubator.nodejs-express.v0.4.3.source.tar.gz
+  templates:
+  - id: simple
+    url: https://github.com/appsody/stacks/releases/download/nodejs-express-v0.4.3/incubator.nodejs-express.v0.4.3.templates.simple.tar.gz
+  - id: scaffold
+    url: https://github.com/appsody/stacks/releases/download/nodejs-express-v0.4.3/incubator.nodejs-express.v0.4.3.templates.scaffold.tar.gz
+  version: 0.4.3
+  ```
+
+Congratulations! You have built your first stack hub index file that defines a set of filtered application stacks from multiple source repositories. This index file should be published to your GitHub repository where it can be accessed by developers. Typically, you would create a Git release for a final version of the index file, and reference the index file from a URL that is similar to `https://github.com/myorg/my-org-repository/releases/latest/download/default-index.yaml`.
```

Comment:
You are not limited to publishing to git, the index just needs to be hosted somewhere that is accessible to the users.

Reply:
Updated